keylime/cmd/create_policy.py

```diff
@@ -502,6 +502,10 @@ def main() -> None:
 
     policy = process_signature_verification_keys(args.ima_signature_keys, policy)
 
+    # Ensure we only have unique values in lists
+    for key in ["digests", "ima-buf", "keyrings"]:
+        policy[key] = {k: sorted(list(set(v))) for k, v in policy[key]}
```

Comment:
my apologies, I write the code without testing, as should be `policy[key].items()` (still not tested)

Reply:
Ooops. Fixed.

Reply:
Support for excludes is still missing completely and once that's being added we can make sure that that list has also only unique values. But for that I'll hold off for another PR.

Reply:
... rebase on latest master and added support for exclude list.